src/main/java/models/Player.java

```diff
@@ -0,0 +1,18 @@
+package models;
+
+public class Player {
+
+    private String name;
+    private int score;
```

Comment:
In the class Player, variables "name" and "score"  are used, while in the class Team you use variables "teamName" and "score". For a bit of clarification and consistency maybe use "teamName" and "playerName" or something similar with "score" as well.